layers/+lang/haskell/README.org

```diff
@@ -373,26 +381,26 @@ This top-level command is prefixed by ~SPC m~:
 | ~SPC m g b~ | return from definition |
 
 * Syntax checking
-At the moment there are four components, which can check the syntax and indicate
+At the moment there are four components, which can indicate
 errors and warnings in the code. Those components are:
-- flycheck
+- dante, intero (via flycheck)
```

Comment:
I am not sure about this line, does this make sense what I wrote here? Flycheck is frontend for syntax checker, and syntax checkers are dante or intero. But maybe somebody else also? Is it wrong to phrase it this way?

Reply:
Nope perfectly fine this way :+1: